### PR TITLE
chore: k8s manifest filters volume

### PR DESCRIPTION
**Pull request by krol3**
k8s manifest adding filters volume

### CODE REVIEW

**Comment by krol3** [comment]
@AndreyLevchenko please review the mount folders in the kubernetes manifest

File: deploy/kubernetes/postee.yaml [hunk 1]
```diff
@@ -187,7 +194,9 @@ spec:
             - name: postee-config
               mountPath: /config
             - name: rego-template
-              mountPath: /server/rego-template/custom
+              mountPath: /server/rego-template
+            - name: rego-filters
+              mountPath: /server/rego-filters
```
Comment (AndreyLevchenko) [inline deploy/kubernetes/postee.yaml:199]: @krol3 
Can users add something custom to this folder?

Comment (krol3) [reply deploy/kubernetes/postee.yaml:199]: If I understand correctly, the user could add more rego samples  https://github.com/aquasecurity/postee/tree/main/rego-filters

Comment (DmitriyLewen) [reply deploy/kubernetes/postee.yaml:199]: Hi @ krol3
We have a mistake: postee is using the rego-template**s** folder.
In addition, if we mount Kubernetes volumes `/server/rego-templates` and `/server/rego-filters`, then these volumes will be without rules by default, which we copy from the docker image. It is required to mount the folders for rego samples in subfolders, for example server `/rego-templates/custom`, after which the user can save and use their rego samples.

Comment (krol3) [reply deploy/kubernetes/postee.yaml]: @DmitriyLewen please take a look to the folders

**Review by AndreyLevchenko — APPROVED** [review]

File: deploy/kubernetes/postee.yaml [hunk 2]
```diff
@@ -188,6 +195,8 @@ spec:
               mountPath: /config
             - name: rego-template
               mountPath: /server/rego-template/custom
```
Comment (DmitriyLewen) [inline deploy/kubernetes/postee.yaml:197]: Use folder with suffix 's':
```suggestion
              mountPath: /server/rego-templates/custom
```

File: deploy/kubernetes/postee.yaml (hunk 1 above)
Comment (krol3) [reply deploy/kubernetes/postee.yaml]: Thanks @DmitriyLewen  I applied your changes.

Comment (krol3) [reply deploy/kubernetes/postee.yaml]: @AndreyLevchenko  could you review this PR? what is missing to merge it?

**Pull request merged**